sdk/servicebus/Azure.Messaging.ServiceBus/tests/Transactions/TransactionLiveTests.cs

```diff
@@ -297,14 +297,7 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
 
                 await receiver.CompleteMessageAsync(receivedMessage1);
 
-                // the service seems to abandon the message that
-                // triggered the InvalidOperationException
-                // in the transaction
-                Assert.That(
-                    async () =>
-                    await receiver.CompleteMessageAsync(receivedMessage2), Throws.InstanceOf<ServiceBusException>()
-                    .And.Property(nameof(ServiceBusException.Reason))
-                    .EqualTo(ServiceBusFailureReason.MessageLockLost));
+                await receiver.CompleteMessageAsync(receivedMessage2);
```

Comment:
Is there any reason to be worried in case some customers might not be currently completing this message in a similar scenario?

Reply:
This is just demonstrating that the message can be completed in such a scenario whereas before it would throw. If someone was completing it and relying on it throwing that would be a break to the behavior, but I think this is unlikely in a real world scenario - it may be possible in tests.

Reply:
Yeah, I guess I was wondering more like if they were relying on the message being abandoned before and not calling complete on the messages that caused exceptions.
I don't think it matters much though because I don't think anyone would be using a transaction that always fails (which I think in this scenario is a deterministic failure).

Reply:
Hmm not sure I follow. How would the message get abandoned before?